### DISCUSSION TITLE
Baileys

### DISCUSSION THREAD

**Discussion by yohanesokta** [deaafrizal/forum#51] [Troubleshooting Room] [answered]
Ada yang pernah pake *wishkeysocket/baileys* untuk whatsapp api , disini saya mempunyai kendala ketika mengambil isi pesan dari **conversation** , terkadang pesan terkirim dengan message type **extendedTextMessage** , kapankah pesan itu bertype **conversation** atau **extendedTextMessage**

**Comment by Fokusdotid** [comment]
Ketika mengirim pesan text :'v

**Reply by yohanesokta** [reply]
yup , tapi beberapa pesan aja

**Comment by ahmadhidayat22** [comment]
kayaknya extendedTextMessage itu pas ketika ada chat dari grup

**Comment by Fokusdotid** [comment, marked as answer]
Sbnrnya mah extendedTextMessage itu muncul di sebabkan pengirim menggunakan more options, makanya type message yg di hasilkan ber type extendedTextMessage

**Reply by yohanesokta** [reply]
Ok termikasih bang
